grails-app/controllers/org/pih/warehouse/api/PutawayApiController.groovy

```diff
@@ -79,6 +80,7 @@ class PutawayApiController {
             putawayItem.availableItems =
                     inventoryService.getAvailableBinLocations(putawayItem.currentFacility, putawayItem.product)
             putawayItem.inventoryLevel = InventoryLevel.findByProductAndInventory(putawayItem.product, putaway.origin.inventory)
+            putawayItem.quantityAvailable = inventoryService.getQuantity(putawayItem.currentFacility.inventory, putawayItem.currentLocation, putawayItem.inventoryItem)
```

Comment:
I'm almost sure that quantity and quantityAvailable will be the same in this case, so you can just use the quantity, there is no need to calculating it again, but you can double check just to be sure